diff -up piper1-gpl-main/libpiper/CMakeLists.txt.omv~ piper1-gpl-main/libpiper/CMakeLists.txt
--- piper1-gpl-main/libpiper/CMakeLists.txt.omv~	2025-10-06 02:24:37.801651116 +0200
+++ piper1-gpl-main/libpiper/CMakeLists.txt	2025-10-06 02:29:45.818455490 +0200
@@ -79,7 +79,7 @@ target_link_libraries(piper
 include(GNUInstallDirs)
 
 install(TARGETS piper
-    DESTINATION ${CMAKE_INSTALL_PREFIX}
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}
 )
 
 install(
@@ -89,14 +89,14 @@ install(
 
 install(
     DIRECTORY ${ONNXRUNTIME_DIR}/lib/
-    DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/piper
 )
 
 # Copy espeak-ng-data
 set(ESPEAKNG_DATA_SRC ${CMAKE_BINARY_DIR}/espeak_ng-install/share/espeak-ng-data)
-set(ESPEAKNG_DATA_DST ${CMAKE_INSTALL_PREFIX}/espeak-ng-data)
+set(ESPEAKNG_DATA_DST ${CMAKE_INSTALL_PREFIX}/share/espeak-ng-data)
 
 install(
     DIRECTORY ${ESPEAKNG_DATA_SRC}
-    DESTINATION ${CMAKE_INSTALL_PREFIX}
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/share
 )
