--- a/libpiper/src/piper.cpp
+++ b/libpiper/src/piper.cpp
@@ -365,12 +365,13 @@
                                                "scales", "sid"};
 
     // Get all output names
-    std::vector<std::string> output_names_strs =
+/*    std::vector<std::string> output_names_strs =
         synth->session->GetOutputNames();
     std::vector<const char *> output_names;
     for (const auto &name : output_names_strs) {
         output_names.push_back(name.c_str());
-    }
+    } */
+    std::array<const char *, 1> output_names = {"output"};
 
     // Infer
     auto output_tensors = synth->session->Run(
