diff -up piper1-gpl-main/src/piper/audio_playback.py.omv~ piper1-gpl-main/src/piper/audio_playback.py
--- piper1-gpl-main/src/piper/audio_playback.py.omv~	2025-10-08 18:27:25.411806493 +0200
+++ piper1-gpl-main/src/piper/audio_playback.py	2025-10-08 18:27:31.085927715 +0200
@@ -24,8 +24,6 @@ class AudioPlayer:
                 "s16le",
                 "-ar",
                 str(self.sample_rate),
-                "-ac",
-                "1",
                 "-",
             ],
             stdin=subprocess.PIPE,
