--- a/src/piper/__main__.py
+++ b/src/piper/__main__.py
@@ -92,8 +92,8 @@
         "--data-dir",
         "--data_dir",
         action="append",
-        default=[str(Path.cwd())],
-        help="Data directory to check for voice models (default: current directory)",
+        default=[str(Path.cwd()), '/usr/share/piper/voices'],
+        help="Data directory to check for voice models (default: current directory and /usr/share/piper/voices)",
     )
     #
     parser.add_argument(
--- a/src/piper/http_server.py
+++ b/src/piper/http_server.py
@@ -56,8 +56,8 @@
         "--data-dir",
         "--data_dir",
         action="append",
-        default=[str(Path.cwd())],
-        help="Data directory to check for downloaded models (default: current directory)",
+        default=[str(Path.cwd()), '/usr/share/piper/voices'],
+        help="Data directory to check for downloaded models (default: current directory and /usr/share/piper/voices)",
     )
     parser.add_argument(
         "--download-dir",
