diff -up piper1-gpl-main/piper-speak/CMakeLists.txt.omv~ piper1-gpl-main/piper-speak/CMakeLists.txt
--- piper1-gpl-main/piper-speak/CMakeLists.txt.omv~	2025-10-06 22:54:36.376452455 +0200
+++ piper1-gpl-main/piper-speak/CMakeLists.txt	2025-10-06 22:55:18.909447950 +0200
@@ -11,10 +11,10 @@ set(PIPER_HEADER ${LIBPIPER_DIR}/../incl
 #libpiper is inside the root of LIBPIPER_DIR already
 #so no need for seperate variable
 set(ONNX_LIB_DIR /usr/lib64)
-set(ESPEAK_NG_DATA=${LIBPIPER_DIR}/espeak-ng-data/)
+set(ESPEAK_NG_DATA /usr/share/espeak-ng-data/)
 
 #We also need the path to the voices: the .onnx and the .onnx.json
-set(VOICES_DIR ${CMAKE_BINARY_DIR}/../download/voices)
+set(VOICES_DIR /usr/share/piper/voices)
 
 ### Actually find the libraries
 
@@ -61,7 +61,7 @@ target_link_libraries(${PROJECT_NAME} PR
 target_link_libraries(${PROJECT_NAME} PRIVATE ${PCAUDIO_LIB})
 target_compile_definitions(${PROJECT_NAME}
 	#tell the code where ESPEAK_NG_DATA is located
-	PRIVATE -DBUILD_ENV_ESPEAK_NG_DATA=\"${ESPEAK_EN_DATA}\"
+	PRIVATE -DBUILD_ENV_ESPEAK_NG_DATA=\"${ESPEAK_NG_DATA}\"
 	#tell the code where the voices are located
 	PRIVATE -DBUILD_ENV_VOICES_PATH=\"${VOICES_DIR}\"
 )
